service/implementation/auto_request_api/__init__.py

```diff
@@ -1,29 +1,41 @@
 import time
 from apscheduler.schedulers.background import BackgroundScheduler
 from concurrent.futures import ThreadPoolExecutor
-from logic_auto_request import apis, auto_request_system
+from service.implementation.auto_request_api.logic_auto_request import apis, auto_request_system, keep_db_alive
```

Comment:
Placing executable code (and especially something like here) is generally a bad idea:

"The __init__.py file is intended for package initialization, not for running or scheduling tasks. It should primarily set up the namespace for a package or perform lightweight configuration."

__init__.py is sort of a header for your package and it gets executed every time this package is imported somewhere. It should describe/prepare the package for correct workflow instead of executing real logic.

I strongly suggest changing this logic